apps-rendering/src/components/LastUpdated/index.tsx

```diff
@@ -21,10 +21,7 @@ const LastUpdated: FC<Props> = ({ lastUpdatedDisplay, lastUpdated }: Props) => (
 			`}
 		`}
 	>
-		<time
-			className="js-last-updated-time"
-			dateTime={lastUpdated.toISOString()}
-		>
+		<time dateTime={lastUpdated.toISOString()}>
```

Comment:
I removed the `js-` class to prevent this value being overwritten by a client side script